charts/spv-wallet-stack/charts/spv-wallet/templates/configmap.yml

```diff
@@ -67,7 +67,12 @@ data:
       webhook_endpoint: "{{ .Values.config.notifications.webhook_endpoint }}"
     paymail:
       beef:
-        block_header_service_url: 'http://{{- printf "%s-block-headers-service" .Release.Name | trunc 63 | trimSuffix "-" }}/api/v1/chain/merkleroot/verify'
+        {{- if .Values.config.paymail.beef.block_headers_service_url }}
+          block_headers_service_url: {{ .Values.config.paymail.beef.block_headers_service_url }}
+        {{- else }}
+          block_headers_service_url: '{{ default (printf "%s-block-headers-service" .Release.Name | trunc 63 | trimSuffix "-") }}/api/v1/chain/merkleroot/verify'
+        {{- end }}
+        block_headers_service_auth_token: {{ .Values.config.paymail.beef.block_headers_service_auth_token }}
         use_beef: {{  .Values.config.paymail.beef.use_beef }}
```

Comment:
Because you add deeper indentation in the if-else statement, those 2 have now wrong indentation

Reply:
fixed